docs/advanced/document-store/redis.md

```diff
@@ -281,6 +273,102 @@ More example filter expresses
 }
 ```
 
+
+(vector-search-index)=
+### Update Vector Search Indexing Schema
+
+Redis vector similarity supports two indexing methods:
+
+- **FLAT**: Brute-force search. 
+- **HNSW**: Efficient and robust approximate nearest neighbor search using Hierarchical Navigable Small World graphs.
+
+Both methods have some mandatory parameters and optional parameters.
+
+```{tip}
+Read more about HNSW or FLAT parameters and their default values [here](https://redis.io/docs/stack/search/reference/vectors/#querying-vector-fields).
+```
+
+You can update the search indexing schema on an existing DocumentArray by setting `update_schema` to `True` and changing your configuratoin parameters.
+
+Consider you store Documents with default indexing method `'HNSW'` and distance `'L2'`, and want to find the nearest vectors to the embedding `[8. 8. 8.]`.
+
+```python
+import numpy as np
+from docarray import Document, DocumentArray
+
+n_dim = 3
+
+da = DocumentArray(
+    storage='redis',
+    config={
+        'n_dim': n_dim,
+        'index_name': 'idx',
+        'distance': 'L2',
+    },
+)
+
+da.extend([Document(id=f'{i}', embedding=i * np.ones(n_dim)) for i in range(10)])
+
+np_query = np.ones(n_dim) * 8
+n_limit = 5
+
+results = da.find(np_query, limit=n_limit)
+
+print('\nEmbeddings Approximate Nearest Neighbours:\n')
+for embedding, score in zip(
+    results.embeddings,
+    results[:, 'scores'],
+):
+    print(f' embedding={embedding},\t score={score["score"].value}')
```

Comment:
is there a specific reason for writing like this? why not
```python
for doc in results:
    print(doc.embedding, doc.score.value)
```

Reply:
I agree it is a way more naturral way to work